bin/scripts/set_additional_api_configs.js

```diff
@@ -0,0 +1,38 @@
+var pluginManager = require('../../plugins/pluginManager.js'),
+    countlyDb = pluginManager.dbConnection(),
+    logger = require('../../api/utils/log');
+
+/**
+ * Get plugins config object
+ * */
+countlyDb.collection('plugins').findOne({}, function(err, pluginsConfig) {
+    if (!pluginsConfig && err) {
```

Comment:
I think you mean `or` here not `and`?
Also maybe beneficial to log err object too